android/src/main/java/tech/bam/RNBraintreeDropIn/RNBraintreeDropInModule.java

```diff
@@ -47,6 +51,35 @@ public void show(final ReadableMap options, final Promise promise) {
 
     DropInRequest dropInRequest = new DropInRequest().clientToken(options.getString("clientToken"));
 
+    if (options.hasKey("googlePay")) {
+      final ReadableMap googlePayOptions = options.getMap("googlePay");
+      if (!googlePayOptions.hasKey("amount")) {
+        promise.reject("NO_GOOGLE_PAY_AMOUNT", "You must provide an amount for Google Pay");
+        return;
+      }
+      if (!googlePayOptions.hasKey("currencyCode")) {
+        promise.reject("NO_GOOGLE_PAY_CURRENCY_CODE", "You must provide an currency code for Google Pay");
+        return;
+      }
+
+      GooglePaymentRequest googlePaymentRequest = new GooglePaymentRequest()
+              .transactionInfo(TransactionInfo.newBuilder()
+                      .setTotalPrice(String.valueOf(googlePayOptions.getDouble("amount")))
+                      .setTotalPriceStatus(WalletConstants.TOTAL_PRICE_STATUS_FINAL)
```

Comment:
Shouldn't that be configurable ?

Reply:
Could you explain more detail please?

Reply:
This can be made in a follow-up PR, but apparently total price status can be changed to estimated or not known. This should be ideally an option in this bridge.

https://developers.google.com/android/reference/com/google/android/gms/wallet/WalletConstants.TotalPriceStatus.html